### PR TITLE
maven and gradle docs for junit 5

### PR DESCRIPTION
**Pull request by tnimni**
gradle is missing execution details, seems like a plugin bug

### CODE REVIEW

File: docs/2.0/integration/junit5.adoc [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+=== Maven
+
+Add the following to your *pom.xml*:
+[source, xml, linenums]
+.pom.xml
+----
+<properties>
+        <allure-junit5.version>2.0-BETA19</allure-junit5.version>
+        <aspectj.version>1.8.10</aspectj.version>
+        <!-- Please note that according to Junit 5 documentation, it will only work with maven surefire-plugin 2.19.1 at this time -->
```
Comment (baev) [inline docs/2.0/integration/junit5.adoc]: I rather prefer remove all other properties, and add this note as a note in doc

File: docs/2.0/integration/junit5.adoc [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+
+        <!-- Encoding -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+</properties>
+
+<dependencies>
+    <dependency>
```
Comment (baev) [inline docs/2.0/integration/junit5.adoc]: only allure-junit5 is required

File: docs/2.0/reporting/gradle-autoconfigure-junit5.adoc [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+dependencies {
+    compile 'io.qameta.allure:allure-gradle:2.4'
+    testCompile(
+            'org.junit.jupiter:junit-jupiter-api:5.0.1',
+            'io.qameta.allure:allure-gradle:2.4'
+    )
+    testRuntime(
+            'org.junit.jupiter:junit-jupiter-engine:5.0.1',
+            'org.junit.vintage:junit-vintage-engine:4.12.1',
+            'org.junit.platform:junit-platform-launcher:1.0.2',
```
Comment (baev) [inline docs/2.0/reporting/gradle-autoconfigure-junit5.adoc]: only `junit-jupiter-engine` is required during test runtime

File: docs/2.0/reporting/gradle-autoconfigure-junit5.adoc [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+sourceCompatibility = 1.8
+
+repositories {
+    jcenter()
+    mavenCentral()
+}
+
+dependencies {
+    compile 'io.qameta.allure:allure-gradle:2.4'
+    testCompile(
```
Comment (baev) [inline docs/2.0/reporting/gradle-autoconfigure-junit5.adoc]: lets use the same name convention for all the deps. 

```groovy
compile a
compile b
testCompile c
testCompile d
```

vs 

```groovy
compile(
    a, 
    b
)
testCompile(
    c,
    d
)
```

File: docs/2.0/reporting/gradle-autoconfigure-junit5.adoc [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    version = '2.4.1'
+    autoconfigure = true
+    aspectjweaver = true
+    aspectjVersion = '1.8.10'
+
+    useJunit5 {
+       version = '2.0-BETA21'
+    }
+
+    downloadLink = 'https://dl.bintray.com/qameta/generic/io/qameta/allure/allure/2.4.1/allure-2.4.1.zip'
```
Comment (baev) [inline docs/2.0/reporting/gradle-autoconfigure-junit5.adoc]: no need to specify link

File: docs/2.0/reporting/gradle-autoconfigure-junit5.adoc [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+[source, bash]
+----
+$ ./gradlew clean test
+----
+
+Allure results will appear in **allure-results** folder under the project root. To generate html report and automatically open it in a web browser, run the following command:
+[source, bash]
+----
+$ allure serve allure-results
```
Comment (baev) [inline docs/2.0/reporting/gradle-autoconfigure-junit5.adoc]: you can use `$ ./gradlew allure` and `$ ./gradlew serve` commands

File: docs/2.0/integration/junit5.adoc (hunk 1 above)
Comment (tnimni) [reply docs/2.0/integration/junit5.adoc]: removed all except aspectj.version, as it also appears in junit4

File: docs/2.0/integration/junit5.adoc (hunk 2 above)
Comment (tnimni) [reply docs/2.0/integration/junit5.adoc]: fixed

File: docs/2.0/reporting/gradle-autoconfigure-junit5.adoc (hunk 5 above)
Comment (tnimni) [reply docs/2.0/reporting/gradle-autoconfigure-junit5.adoc]: removed

File: docs/2.0/reporting/gradle-autoconfigure-junit5.adoc (hunk 3 above)
Comment (tnimni) [reply docs/2.0/reporting/gradle-autoconfigure-junit5.adoc]: removed all the rest

File: docs/2.0/reporting/gradle-autoconfigure-junit5.adoc (hunk 4 above)
Comment (tnimni) [reply docs/2.0/reporting/gradle-autoconfigure-junit5.adoc]: changed convention

File: docs/2.0/reporting/gradle-autoconfigure-junit5.adoc (hunk 6 above)
Comment (tnimni) [reply docs/2.0/reporting/gradle-autoconfigure-junit5.adoc]: i changed the command to ./gradlew allureServe as it is in junit 4 docs, as when running allure or serve there's a message that task doesn't exist

**Comment by eroshenkoam** [comment]
ty! 🥇

**Comment by tnimni** [comment]
@eroshenkoam i'll upload another fix later, need to add it to the side menu, i forgot

**Pull request merged**